indico/util/marshmallow.py

```diff
@@ -500,7 +499,7 @@ def _deserialize(self, value, attr, data, **kwargs) -> str | None:
         return super()._deserialize(value, attr, data, **kwargs).lower()
 
 
-class NoneValueEnumField(EnumField):
+class NoneValueEnumField(fields.Enum):
```

Comment:
Might be worth to rename this to `NoneValueEnum` to keep consistent naming now that we have `Enum` instead of `EnumField`, but maybe let's save it for a followup PR to keep the diff smaller

Reply:
personally I like `Field` at the end of custom MM field types, because otherwise the names are often misleading. for base MM fields you usually use them as `fields.Something` so it's clearer. But custom fields are typically imported, so `NoneValueEnumField` is clearer than `NoneValueEnum` (which i'd expect to be some kind of enum).

Same with `ModelField` vs `Model` - from the latter nobody expect a MM field.